app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -71,17 +71,17 @@ def sync_action!
       end
 
       def plan_create_action
```

Comment:
since now all the methods (plan_{create,update,destroy}_action) look the same, i guess we have just one `plan_hook_action` and use it for all `after_create`, `after_update` and `after_destroy`